Graphql package version conflict

Version of FoalTS: v1 as well as v2

I'm getting this error when I try to do an `npm i`:

```sh
npm ERR! While resolving: @foal/graphql@1.12.2
npm ERR! Found: graphql@15.8.0
npm ERR! node_modules/graphql
npm ERR!   graphql@"^15.5.1" from the root project
npm ERR!   peer graphql@"^0.13.0 || ^14.0.0 || ^15.0.0" from @graphql-inspector/core@2.9.0
npm ERR!   node_modules/@graphql-inspector/core
npm ERR!     dev @graphql-inspector/core@"^2.6.1" from the root project
npm ERR!   15 more (@graphql-tools/graphql-file-loader, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer graphql@"^14.3.0" from @foal/graphql@1.12.2
npm ERR! node_modules/@foal/graphql
npm ERR!   @foal/graphql@"^1.12.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: graphql@14.7.0
npm ERR! node_modules/graphql
npm ERR!   peer graphql@"^14.3.0" from @foal/graphql@1.12.2
npm ERR!   node_modules/@foal/graphql
npm ERR!     @foal/graphql@"^1.12.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

I've tracked this issue down to the fact that the `type-graphql` package depends on `graphql@15` whereas `@foal/graphql` is still on 14. Is there any reason we couldn't bump `@foal/graphql` up to v15?

--- a/docs/docs/api-section/graphql.md
+++ b/docs/docs/api-section/graphql.md
@@ -29,6 +29,8 @@ title: GraphQL
 To use GraphQL with FoalTS, you need to install the packages `graphql` and `@foal/graphql`. The first one is maintained by the GraphQL community and parses and resolves queries. The second is specific to FoalTS and allows you to configure a controller compatible with common GraphQL clients ([graphql-request](https://www.npmjs.com/package/graphql-request), [Apollo Client](https://www.apollographql.com/docs/react/), etc), load type definitions from separate files or handle errors thrown in resolvers.
 
 ```bash
+npm install graphql@15 @foal/graphql
+# OR
 npm install graphql@14 @foal/graphql
 ```
 
@@ -418,6 +420,17 @@ export class ApiController extends GraphQLController {
 ```
 
 ## Using TypeGraphQL
+
+
+:::info
+
+TypeGraphQL requires version 15 of the `graphql` package:
+
+```bash
+npm install graphql@15
+```
+
+:::
 
 > *[TypeGraphQL](https://typegraphql.com/) is a library that allows you to create GraphQL schemas and resolvers with TypeScript classes and decorators.*
 

--- a/packages/graphql/package.json
+++ b/packages/graphql/package.json
@@ -50,16 +50,16 @@
     "apollo-cache-inmemory": "~1.5.1",
     "apollo-client": "~2.5.1",
     "apollo-link-http": "~1.5.14",
-    "graphql": "~14.6.0",
+    "graphql": "~15.8.0",
     "graphql-request": "~1.8.2",
     "graphql-tag": "~2.10.1",
     "mocha": "~8.3.0",
     "rimraf": "~2.6.2",
     "ts-node": "~9.0.0",
-    "type-graphql": "~0.17.5",
+    "type-graphql": "~1.1.1",
     "typescript": "~4.0.2"
   },
   "peerDependencies": {
-    "graphql": "^14.3.0"
+    "graphql": ">= 14.3.0 < 16.0.0"
   }
 }

